HeliosCLI/Makefile

```diff
@@ -107,7 +126,8 @@ FORCE:
 
 # generic clean target
 clean:
-	@$(RM) $(DFILES) $(OBJS) $(TARGETS) $(TESTS)
+	@$(RM) -f $(DFILES) $(OBJS) $(FINAL_TARGET) $(TESTS)

```

Comment:
This is kinda pointless if everything is in the build dir and you just rm rf the dir right after this